scripts/nm-quick.sh

```diff
@@ -281,49 +284,46 @@ install_dependencies() {
 
 	OS=$(uname)
 	if [ -f /etc/debian_version ]; then
-		dependencies="git wireguard wireguard-tools dnsutils jq docker.io docker-compose"
+		dependencies="git wireguard wireguard-tools dnsutils jq docker.io certbot docker-compose python-is-python3"
```

Comment:
python only required on debian?

Reply:
Debian / Ubuntu have `python2` & `python3`. `python-is-python3` is just a symlink pkg.

It's needed for the ZeroSSL certbot wrapper, while the regular python package is a dep of certbot.

Reply:
Just check Alpine and it also causes problems. Will use certbot via docker instead.